doc/boot_protocol.rst

```diff
@@ -0,0 +1,274 @@
+Security Kernel Loader
+######################
+
+The Security Kernel Loader (SKL) is an implementation of a Secure Loader for
+the SKINIT CPU instruction. The SKL loader takes a minimal approach relying on
+the DRTM Preamble, e.g. the system bootloader, to load into memory and
+configure the Security Kernel that the SKL will measure and execute.
+
+Memory Layout
+*************
+
+The DRTM Preamble is responsible for allocating a contiguous block of physical
```

Comment:
It makes things simpler otherwise we would have to expand the header each time we add another section that the boot loader could allocate independently. This way, the bootloader just needs to find a single chunk in memory layout table and then the SKL can dice it up however it needs.